### ISSUE TITLE
[4.x] CLI create user password validation

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#37629]
### Steps to reproduce the issue
create a user using the cli
`php cli/joomla.php user:add`

follow the questions to create a user.

```
Add user
========

 Please enter a username:
 > ss

 Please enter a name (full name of user):
 > ss

 Please enter an email address:
 > ss@ss

 Please enter a password:
 > ss
```

### Expected result
minimum validation rules apply

### Actual result
possible to create a user with a 1 character password

**Comment by zero-24** [comment]
Yes looks like there is no validation at all here: https://github.com/joomla/joomla-cms/blob/4.1-dev/libraries/src/Console/AddUserCommand.php#L130

Not sure whether it makes sense to just hardcode the validation there but maybe better implement the checks for the existing validation rules and XML files. Or is there a reason to specificly not implement the xml validation rules via the CLI?

**Comment by brianteeman** [comment]
not sure if I care if it follows the exact site defined rules or just something a little more than just a single character

**Comment by PhilETaylor** [comment]
That's what happens when @joomla/security team does no work for 2 years... https://github.com/joomla/joomla-cms/issues/29817

**Comment by alikon** [comment]
please check #48343
